.github/workflows/integration.yaml

```diff
@@ -209,6 +209,9 @@ jobs:
 
   e2e-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        k8s: [v1.28.13, v1.29.8, v1.30.4, v1.31.1]
```

Comment:
What about renaming `k8s` to `K8S_VERSION`?

Reply:
Hmm I'm not sure if the naming of this matrix value matters too much here since it's going into `KIND_K8S_VERSION` anyway. I just took inspiration from https://github.com/apache/yunikorn-k8shim/blob/master/.github/workflows/pre-commit.yml#L50 when checking what other projects were doing

Reply:
Never mind, it's just a nit, I would prefer a more meaningful name.

Reply:
It's all good, meaningful variable names are a fair nit. Changing to `k8s_version`.